lib/docparser.js

```diff
@@ -401,7 +402,30 @@ YUI.add('docparser', function (Y) {
                 target[tagname] = result;
 
             },
-            'throws': 'return',
+
+            // @throws {type} description
+            'throws': function (tagname, value, target) {
```

Comment:
I *think* you can do `'throws': 'return'` since it's the same implementation. See https://github.com/okuryu/yuidoc/blob/throws/lib/docparser.js#L586-L589